.fixtures.yml

```diff
@@ -6,3 +6,4 @@ fixtures:
     sshkeys_core:
       repo: https://github.com/puppetlabs/puppetlabs-sshkeys_core.git
       puppet_version: ">= 6.0.0"
+    accounts: https://forge.puppet.com/modules/puppetlabs/accounts
```

Comment:
This must also be added in `metadata.json` since the module can't function without it.

Reply:
https://github.com/nod0n/puppet-sftp_jail/commit/b63f8a965988c37af0d8aa9e32f80af3675b6494